### PR TITLE
Fix Russian translation

### PR DESCRIPTION
**Pull request by sooobus**
Changes in this pull request:

- Correct several typos across the tutorial
- Correct idioms usage. Replaced "compare apples with oranges" and "works like a charm" with Russian equivalent instead of word-by-word translation. Note that the latter has negative connotations in Russian. As no one uses "works like a charm" as an idiom, it sounds like coding is magic.
- Removed paragraph with duplicate information in template_extending
- Rewrote the paragraph "Compare things" in order to get rid of using Russian equivalent of "thing". No one uses "thing" and "compare" together in the context of objects comparison. It improves language style and makes text easier to read.

### CODE REVIEW

File: ru/deploy/README.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ $ pa_autoconfigure_django.py https://github.com/<your-github-username>/my-first-
...
 
-Главное, на что нужно обратить внимание сейчас, — это то что твоя база данных на PythonAnywhere никак не связана с базой данных на твоём компьютере. Поэтому там будут разные посты и разные аккаунты администраторов. Как следствие, для базы на PythonAnywhere необходимо создать аккаунт администратора так же, как ты это делала у себя локально с помощью команды `createsuperuser`. На PythonAnywhere заранее активировано вирутальное окружение, так что всё, что тебе нужно сделать — это запустить в консоли PythonAnhywhere команду:
+Главное, на что нужно обратить внимание сейчас, — это то что твоя база данных на PythonAnywhere никак не связана с базой данных на твоём компьютере. Поэтому там будут разные посты и разные аккаунты администраторов. Как следствие, для базы на PythonAnywhere необходимо создать аккаунт администратора так же, как ты это делала у себя локально с помощью команды `createsuperuser`. На PythonAnywhere заранее активировано виртуальное окружение, так что всё, что тебе нужно сделать — это запустить в консоли PythonAnhywhere команду:
```
Comment (telezhnaya) [inline ru/deploy/README.md]:
```suggestion
Главное, на что нужно обратить внимание сейчас, — это то, что твоя база данных на PythonAnywhere никак не связана с базой данных на твоём компьютере. Поэтому там будут разные посты и разные аккаунты администраторов. Как следствие, для базы на PythonAnywhere необходимо создать аккаунт администратора так же, как ты это делала у себя локально с помощью команды `createsuperuser`. На PythonAnywhere заранее активировано виртуальное окружение, так что всё, что тебе нужно сделать — это запустить в консоли PythonAnhywhere команду:
```

File: ru/python_introduction/README.md [hunk 2]
```diff
@@ -547,7 +547,7 @@ False
 -   `1 != 2`
 
 Поздравляем! Логические значения — одна из самых классных фишек программирования, и ты только что научилась ими пользоваться!
-
+ы
```
Comment (telezhnaya) [inline ru/python_introduction/README.md]: кажется, это было лишним :)

Comment (telezhnaya) [inline ru/python_introduction/README.md]:
```suggestion
```

File: ru/template_extending/README.md [hunk 3]
```diff
@@ -118,11 +118,6 @@ blog
     {% endfor %}
 {% endblock %}
 ```
```
Comment (telezhnaya) [inline ru/template_extending/README.md]: Ты уверена, что стоило это удалять?

File: ru/python_introduction/README.md (hunk 2 above)
Comment (sooobus) [reply ru/python_introduction/README.md]: thanks! yes :)

**Comment by magul** [comment]
@sooobus Thanks for your fixes here. Can You also, please, apply them to our Crowdin project here: https://crowdin.com/project/django-girls-tutorial/ru#

File: ru/template_extending/README.md (hunk 3 above)
Comment (sooobus) [reply ru/template_extending/README.md]: Да. Посмотри строки ниже, там повторяется то же самое, но с объяснением. В английской версии этой строки нет.

**Comment by sooobus** [comment]
@magul Can you please clarify if there is an automatic way to do this? Thanks.

**Comment by magul** [comment]
@sooobus The issue here is that we have an automatic way to sync source language (English) to Crowdin and it's already working and we have a semi-automatic way to sync whatever target language from Crowdin here. Translations made on GitHub have their own limitation (there's no way to figure out, what should be updated to be on top of Enligsh version), which are somehow solved by Crowdin.

Already Crowdin Russian translation hits 90% coverage and as soon as it will fully covered I will (or @das-g, or anybody that has Crowdin manager permission) create a branch, that will contain the whole translation and whenever something changes in source version it will be clearly visible there.

**Comment by telezhnaya** [comment]
@magul could you please help us to find these 10% that we still need to translate? We have a guess that we translated everything.

**Comment by magul** [comment]
@telezhnaya most probably You did here, but it's not syncing with Crowdin. On Crowdin there's 87% (actually) coverage:
ru-coverage

To feel that gap there You need to go to: https://crowdin.com/project/django-girls-tutorial/ru# and (unfortunatelly) manually add proper translation to files that are not fully covered.

**Comment by telezhnaya** [comment]
@magul I am afraid that crowdin version is super outdated. It's not only about current changes, we have to rewrite **all** tutorial. Maybe it would be easier to delete crowdin version and re-add the version from github?

**Comment by magul** [comment]
@telezhnaya I'm not a Russian language speaker, so I'm not able to assess the quality of Russian translation there. 87% coverage is actually calculated against the most up-to-date English version.

I could blindly assume that You're right about poor quality there but in the end, Crowdin provide few tools that we need to make the translation process more manageable and it would be great to have Russian language there in proper shape. So I'm just kindly asking, to help also there :)

Also if You want to help there @telezhnaya I can make You a proofreader there and You should be able to delete translations per sentence, but don't know how this whole language deletion would look like (not mentioning that will be super rude for these few people that improved greatly coverage there in past few months).

**Comment by telezhnaya** [comment]
@magul no no no, I meant that Github has proper version now, and huge amount of work were done in past few months to update the tutorial. We (organizers and mentors) decided to update the tutorial before the event in Saint Petersburg. 

I meant that we need to re-set crowdin version, and take Github version as the main one for now.

I would be so proud if you add me to Django Girls team in Github!

**Comment by magul** [comment]
@telezhnaya I spotted that work here, but in parallel, there was work done on Crowdin also. And in the end, it would be way more manageable to maintain translation there rather than here.

Regarding GitHub membership, I'm not in the position to make such arrangement but will ask @RachellCalhoun on Slack to add You to Russian PR team.

**Comment by telezhnaya** [comment]
@magul I need to tell you more about our processes.

In December 2018 we (organisers and mentors of Django Girls Saint Petersburg) realised that Russian translation of the tutorial was still using Django 1.1 (while the whole world use 2.0, including English version of DG tutorial). And we decided to update the translation as accurate as it was possible.

We organised the translation of the entire text, split it between all volunteers, translated it. Some of us (at least @patiencedaur ) were trying to use Crowdin, and all of them had different types of issues. At least, there was the issue that the Crowdin version was outdated even at the moment when we decided to start our work. That's why we decided to publish all work in Github, including transferring pieces from Crowdin.

So, at the current moment, all our team can say that Github has the most fresh and up-to-date version. That was our big goal before our event in March, 8th.

It would be a waist of time if we will transfer all our work to Crowdin. What we really want to do is just to copy Github version to Crowdin, and then configure automatic update so that these 2 versions could be updated automatically.

**Comment by das-g** [comment]
> What we really want to do is just to copy Github version to Crowdin, and then configure automatic update so that these 2 versions could be updated automatically.

We would very much like a way to automatically sync GitHub &rarr; Crowdin, too, as it's not uncommon that improvements for non-English versions (i.e. for translations) are contributed on GitHub instead of Crowdin. Unfortunately, we don't know of any way to implement that, yet.

Maybe we should contact Crowdin support and ask them what they suggest in this situation?

**Comment by telezhnaya** [comment]
@das-g maybe it's easier just to say that non-English versions are developed via Github only? It will reduce the number of issues dramatically :-)

In the end, Github has convenient and familiar interface for all developers, and we don't even need to fork and download the project to contribute to it. I fixed some minor issues in @sooobus pull request just from my browser.

If you disagree with me, it has sense to contact Crowdin support trying to solve our issues.

Thank you!

**Comment by das-g** [comment]
In the beginning, I struggled with contributing through Crowdin, too. But when lots of changes happen to the English original, it's much easier to see on Crowdin what needs to be re-translated to come in sync with the English version again. That, and the translation memory that helps to translate sentences similar to some that already have been translated, are probably the main reasons to use Crowdin.

**Comment by magul** [comment]
@telezhnaya
> Some of us (at least @patiencedaur ) were trying to use Crowdin, and all of them had different types of issues.

Would love to hear, what kind of issue You had there.

> At least, there was the issue that the Crowdin version was outdated even at the moment when we decided to start our work.

AFAIK Crowdin holds the most current English version. It's usually synced from GitHub in a minute or so. Regarding Russian version it could be out-of-date (however it's almost imposible that it would be more out-of-date than Django 1.11 as it's translated in 87% there) , because changes are made here and not applied on Crowdin (that's why I'm politely asking for that at every pull request targeting translations here).

> That's why we decided to publish all work in Github, including transferring pieces from Crowdin.

And that's super fine, but it will be difficult to maintain it here. I agree with @das-g that in long run there are tools on Crowdin that allow You to maintain translation in less time than doing that on GitHub.

> @das-g maybe it's easier just to say that non-English versions are developed via Github only? It will reduce the number of issues dramatically :-)

That's discussion was conducted around 2 years ago (between @RachellCalhoun, @aniav and me) and after hearing from feedback translators community we decided to left Crowdin in place (we can open that discussion once again of course, but since that time we were able to make 12 or so languages up-to-date so it will be difficult to abandon it now). As You can see, we're accepting GitHub pull request, but in long run there are clear benefits of having more sophisticated translation system than GitHub pull requests, which I and @das-g see.

**Pull request merged**